pom.xml

```diff
@@ -1000,10 +1000,20 @@
 							<goal>install-node-and-yarn</goal>
 						</goals>
 						<configuration>
-							<nodeVersion>v16.15.1</nodeVersion>
+							<nodeVersion>v16.16.0</nodeVersion>
 							<yarnVersion>v1.22.19</yarnVersion>
 						</configuration>
 					</execution>
+					<!-- update yarn to latest version -->
+					<execution>
+						<id>yarn version</id>
+						<goals>
+							<goal>yarn</goal>
+						</goals>
+						<configuration>
+							<arguments>set version stable</arguments>
```

Comment:
I'm not sure about this step since it will slow down builds. When the web settings branch is merged to main, Renovate will keep yarn updated (that is actually a main reason I started using Renovate instead of Dependabot)